src/app/component/ProductCard/ProductCard.style.scss

```diff
@@ -10,9 +10,12 @@
  */
 
 :root {
-    --product-card-picture-mask-color: #f5f5f5;
-    --product-card-color-border-color: #efefef;
+    --product-card-picture-mask-color: rgb(245, 245, 245);
+    --product-card-color-border-color: rgb(239, 239, 239);
+    --product-card-text-border-color: rgb(151, 151, 151);
     --product-cart-picture-label-background: rgba(255, 255, 255, .7);
+    --product-cart-reviews-background: rgba(243, 243, 243, .55);
+    --product-cart-WishListButton-background: rgba(243, 243, 243, .78);
```

Comment:
I am terribly sorry, indeed we use hash notation. 
Please change this back (and I believe your code has another place where you use colors, which I made my falsy judgment from).

Reply:
Also, here it is contextually better to use `wishlist-button` over `WishListButton`